CMakeLists.txt

```diff
@@ -641,7 +641,6 @@ target_link_libraries(
           littlefs
           lpng
           STM32::USBHost
-          CPU
```

Comment:
Removed and fixed.
I also changed linkage of osGetCPUUsage() to c++, changed its return type to int to avoid casting it from `uint32_t` to `uint16_t` and then to `int`, renamed the header to hpp, improved documentation of configurable parameters and removed redundant static variable in GUI.